### PR TITLE
Pre 1.2.0 

### PR DESCRIPTION
**Pull request by ap891843**

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -459,18 +459,18 @@
...
                     "type": "string",
                     "enum": [
                         "JAVA",
                         "NATIVE"
                     ],
                     "default": "JAVA",
                     "enumDescriptions": [
-                        "JAVA  - Run java Server",
-                        "NATIVE - Run native Server"
+                        "JAVA  - Run from a Java ARchive (jar). Requires java runtime to be installed",
```
Comment (ap891843) [inline clients/cobol-lsp-vscode-extension/package.json:470]: @zeibura is this ok?

File: clients/cobol-lsp-vscode-extension/package.json [hunk 2]
```diff
@@ -459,18 +459,18 @@
...
+                        "NATIVE - Use platform binary"
                     ],
-                    "description": "COBOL LS server type."
+                    "description": "COBOL LS server Runtime."
```
Comment (ap891843) [inline clients/cobol-lsp-vscode-extension/package.json:473]: @zeibura is this ok?

File: server/engine/src/main/resources/resourceBundles/messages_en.properties [hunk 3]
```diff
@@ -87,7 +87,7 @@ variables.integer=integer
 variables.blankWhenZero=BLANK WHEN ZERO
 variables.signClause=SIGN CLAUSE
 variables.mapName=Map name
-workspaceError.ServerType=Server type NATIVE does not support dialects.
+workspaceError.ServerType=Ensure that you have Java installed and that your serverRuntime is set to JAVA for dialect support.
```
Comment (ap891843) [inline server/engine/src/main/resources/resourceBundles/messages_en.properties:90]: @zeibura

File: clients/cobol-lsp-vscode-extension/package.json (hunk 2 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/package.json:473]: runtime (no capital)

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/package.json:470]: get rid of the capital R in ARchive

File: clients/cobol-lsp-vscode-extension/package.json (hunk 2 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json]: updated

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json]: updated

**Review by Nurkambay — APPROVED** [review]

**Pull request merged**